sycl/test/abi/sycl_symbols_linux.dump

```diff
@@ -3619,6 +3619,7 @@ _ZN4sycl3_V113aligned_allocEmmRKNS0_5queueENS0_3usm5allocERKNS0_13property_listE
 _ZN4sycl3_V113aligned_allocEmmRKNS0_5queueENS0_3usm5allocERKNS0_6detail13code_locationE
 _ZN4sycl3_V113aligned_allocEmmRKNS0_6deviceERKNS0_7contextENS0_3usm5allocERKNS0_13property_listERKNS0_6detail13code_locationE
 _ZN4sycl3_V113aligned_allocEmmRKNS0_6deviceERKNS0_7contextENS0_3usm5allocERKNS0_6detail13code_locationE
+_ZN4sycl3_V113is_compatibleERKSt6vectorINS0_9kernel_idESaIS2_EERKNS0_6deviceE
```

Comment:
If I am not mistaken you need to update windows symbols too, otherwise you'll get a failure in post-commit.

Reply:
Working on this

Reply:
Done [8464034](https://github.com/intel/llvm/pull/7510/commits/8464034b944a709c3750f72b417122703f3dcbe4)